numba/openmp.py

```
@@ -4330,7 +4348,8 @@ def target_teams_distribute_directive(self, args):
         self.some_target_directive(args, "TARGET.TEAMS.DISTRIBUTE", 3, has_loop=True)
 
     def target_teams_loop_directive(self, args):
-        self.some_target_directive(args, "TARGET.TEAMS.DISTRIBUTE.PARALLEL.LOOP.SIMD", 3, has_loop=True)
+        self.some_target_directive(args, "TARGET.TEAMS.DISTRIBUTE.PARALLEL.LOOP", 3, has_loop=True)
```

Add also an alias definition to redirect  `TARGET.TEAMS.DISTRIBUTE.PARALLEL.LOOP.SIMD` to `TARGET.TEAMS.DISTRIBUTE.PARALLEL.LOOP`. Do we have a test?

---

Are you suggesting we silently drop simd in "target teams distribute parallel for simd"?

Need to add a test I think but I'm losing track of which combinations do and don't have tests.

---

The test_openmp.py has tests with combinations, naming should help to grep

---

Redirect `target_teams_distribute_parallel_for_simd` to `target_teams_distribute_parallel_for`

---

Done.